apps/web/components/blog/footer/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+
+import Link from "next/link";
+import { useTranslation } from "react-i18next";
+
+import { Container } from "@repo/cms/container";
+
+export const Footer = () => {
+  const { t } = useTranslation();
+
+  return (
+    <footer className="border-t-color border-gray200 mt-10 border-t">
+      <Container className="py-8">
+        <h2 className="h4 mb-4">{t("footer.aboutUs")}</h2>
+        <div className="max-w-4xl">{t("footer.description")}</div>
+        <div className="mt-8">
```

Comment:
i would already link back to https://www.jan-ingenhousz-institute.org/ in the footer